R/utils-classes.R

```diff
@@ -66,21 +66,21 @@ print.bcdc_promise <- function(x, ...) {
 
 #' @export
 print.bcdc_record <- function(x, ...) {
-  cat("B.C. Data Catalogue Record:\n   ", x$title, "\n")
+  cat("B.C. Data Catalogue Record:", x$title, "\n")
   cat("\nName:", x$name, "(ID:", x$id, ")")
   cat("\nPermalink:", paste0("https://catalogue.data.gov.bc.ca/dataset/", x$id))
   cat("\nSector:", x$sector)
   cat("\nLicence:", x$license_title)
   cat("\nType:", x$type)
   cat("\nLast Updated:", x$record_last_modified, "\n")
-  cat("\nDescription:\n")
-  cat(paste0("    ", strwrap(x$notes, width = 85), collapse = "\n"), "\n")
+  cat("\nDescription:", paste0(strwrap(x$notes, width = 85), collapse = "\n"), "\n")
 
   cat("\nResources: (", nrow(x$resource_df), ")\n")
+  print(x$resource_df)
```

Comment:
You could also do `print(bcdc_tidy_resources(x))`... this will make sure that if we change the name/structure of a record in the future, we only have to change how we access the resource df once (in `bcdc_tidy_resources()`)